### ISSUE TITLE
Build fails due to lowercase `makefile`

### ISSUE THREAD

**Issue by cc-michaelu** [dwmkerr/linux-kernel-module#6]
Hello,

The build is broken due to the makefile being lowercase.
```
michael_uman@ThinkPad-X1-Carbon-7th ~/gitroot/linux-kernel-module/greeter (main)$ make
make -C /lib/modules/5.11.0-40-generic/build/ M=/home/michael_uman/gitroot/linux-kernel-module/greeter modules
make[1]: Entering directory '/usr/src/linux-headers-5.11.0-40-generic'
scripts/Makefile.build:44: /home/michael_uman/gitroot/linux-kernel-module/greeter/Makefile: No such file or directory
make[2]: *** No rule to make target '/home/michael_uman/gitroot/linux-kernel-module/greeter/Makefile'.  Stop.
make[1]: *** [Makefile:1849: /home/michael_uman/gitroot/linux-kernel-module/greeter] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-5.11.0-40-generic'
make: *** [makefile:8: default] Error 2
```

Changing `makefile` to `Makefile` fixes the problem for me.

```
Linux ThinkPad-X1-Carbon-7th 5.11.0-40-generic #44~20.04.2-Ubuntu SMP Tue Oct 26 18:07:44 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
michael_uman@ThinkPad-X1-Carbon-7th ~/gitroot/linux-kernel-module.old/greeter (master)$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.3 LTS
Release:	20.04
Codename:	focal


```

**Comment by cc-michaelu** [comment]
File PR to resolve issue:

https://github.com/dwmkerr/linux-kernel-module/pull/7

**Comment by dwmkerr** [comment]
Resolved in #7 - thank-you @cc-michaelu - I didn't realise that it was a hard requirement on some environments to capitalise the `M` in the makefile - useful learning for me!
